app/models/user.rb

```diff
@@ -184,6 +184,7 @@ class User < ApplicationRecord
   has_many :tweets, dependent: :nullify
   has_many :webhook_endpoints, class_name: "Webhook::Endpoint", inverse_of: :user, dependent: :delete_all
   has_many :devices, dependent: :delete_all
+  has_many :sponsorships, dependent: :destroy
```

Comment:
this mirrors https://github.com/forem/forem/blob/master/app/models/organization.rb#L33